.github/actions/get-image-tag/action.yml

```diff
@@ -11,10 +11,13 @@ runs:
   steps:
     - name: Get docker image tag name
       shell: bash
+      env:
+        BRANCH_NAME: ${{ inputs.branch_name }}
+        RUN_ID: ${{ github.run_id }}
       run: |
-        if [[ "${{ inputs.branch_name }}" == "master" ]] || [[ "${{ inputs.branch_name }}" == release/* ]]; then
+        if [[ "$BRANCH_NAME" == "master" ]] || [[ "$BRANCH_NAME" == release/* ]]; then
```

Comment:
Why do we need those changes?

Reply:
@MichalKinas because of failing sonar after each of our merge: 
https://github.com/Alfresco/alfresco-content-app/commits/develop/

Reply:
Sonar says it's blocker security issue